assets/src/edit-story/components/canvas/page.js

```diff
@@ -25,6 +25,7 @@ const Background = styled.div.attrs( { className: 'container' } )`
 
 function Page() {
 	const [ targetEl, setTargetEl ] = useState( null );
+	const [ targetRefs, setTargetRefs ] = useState( [] );
```

Comment:
We already have `setNodesById` and also the new story model includes element handles. I think we can just use `setNodesById` for now until the new story model is in use. I also used it here: https://github.com/ampproject/amp-wp/pull/4018/files#diff-5f63d958b23bfdcfd7a3491bbba69eb9R27. Overall that PR changes a number of things for managing element/node mapping, so it might be easier to get that in first and focus this PR on group transformer logic?

Reply:
There's a [previous commit](https://github.com/ampproject/amp-wp/commit/597c4a2acfb22c86457658235d0ee44f4b7722e3) with `setNodesById`, however, as targets we'd need the child of the nodes that are mapped there, it didn't work as expected when testing, however, perhaps something has changed or perhaps the changes done in #4018 will make it possible.

Sure, let's wait until that PR is merged and then take a look again.

Reply:
Yeah, #4018 is probably not an end-point here. But I do think we'd want to simplify our id/node mapping. I still feel like we'll end up making our wrapper element stronger and that should give us more options.